synapse/handlers/federation.py

```diff
@@ -99,37 +99,6 @@
 )
 
 
-def get_domains_from_state(state: StateMap[EventBase]) -> List[Tuple[str, int]]:
```

Comment:
Eliminating this option to get domains for a room in favor of `get_current_hosts_in_room`.